doc/classes/TabBar.xml

```diff
@@ -414,6 +414,7 @@
 		</theme_item>
 		<theme_item name="tab_hovered" data_type="style" type="StyleBox">
 			The style of the currently hovered tab. Does not apply to the selected tab.
+			[b]Note:[/b] This style will be drawn with the same width as [theme_item tab_unselected] at minimum. This is done to avoid an infinite loop when hovering between two tabs with the cursor.
```

Comment:
I don't think that there needs to be a rationale added to the docs. The note can just state the fact, it's not like the reasoning changes anything for the user who reads it.

Reply:
I personally think it's good to be thorough here, just in case the user asks: "OK, but why?".

Reply:
We can wait until someone asks that. Besides, it kind of reads like "There is a bug if we don't do this, and we don't have a better solution available".

Reply:
> There is a bug if we don't do this, and we don't have a better solution available.

Probably because that's exactly what this is. :no_mouth:

Reply:
Yeah, but putting this into the docs won't save you soul and won't offer you penance 😛

Reply:
Fine, fine, it has been removed. :man_shrugging: